packages/salesforcedx-vscode-replay-debugger/src/states/variableBeginState.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { ApexVariable } from '../adapter/apexReplayDebug';
+import { LogContext } from '../core/logContext';
+import { DebugLogState } from './debugLogState';
+
+export class VariableBeginState implements DebugLogState {
+  private fields: string[];
+
+  constructor(fields: string[]) {
+    this.fields = fields;
+  }
+  public handle(logContext: LogContext): boolean {
+    const currFrame = logContext.getTopFrame();
+    if (currFrame) {
+      const id = currFrame.id;
+      const frameInfo = logContext.getFrameHandler().get(id);
+      const name = this.fields[3];
```

Comment:
In the case of a static variable this will be "ClassName.VariableName".
In the case of a local variable this will be "variableName"